.mdformat.toml

```diff
@@ -0,0 +1,2 @@
+end_of_line = "lf"
+exclude = [".tox/**", ".venv/**", "**/*.m[!d]", "**/*.[!m]d"]
```

Comment:
won't `exclude` result in an error on `python<3.13`?

Reply:
... because if so, then this probably should be commented out until hukkin/mdformat#489 is implemented